script.js

```diff
@@ -1,4 +1,147 @@
+    currentGameMode = "Player 1";
+  }
+
+  // round 1 game - player 1
+  // 1. player 1 roll dice
+  if (currentGameMode == "Player 1") {
+    diceRoll1.push(rollDice());
+    diceRoll2.push(rollDice());
+    myOutputValue =
+      userName1 +
+      ", You rolled " +
+      diceRoll1[0] +
+      " for dice 1 and " +
+      diceRoll2[0] +
+      " for dice 2. <br><br> Please choose the order of the dice. ";
+    currentGameMode = "Player 1 order";
+  }
+  // 2. player 1 choose which order & prompt player 2 to roll
+  else if (currentGameMode == "Player 1 order" && input == 1) {
+    var player1DiceOrder = diceRoll1[0].toString() + diceRoll2[0].toString();
+    player1Num = Number(player1DiceOrder);
+    console.log("player1Num");
+    console.log(player1Num);
+    myOutputValue = `Your number is ${player1DiceOrder}. <br><br>${userName2}, Please press submit to roll your dice`;
+    currentGameMode = "Player 2";
+  } else if (currentGameMode == "Player 1 order" && input == 2) {
+    var player1DiceOrder = diceRoll2[0].toString() + diceRoll1[0].toString();
+    player1Num = Number(player1DiceOrder);
+    myOutputValue = `Your number is ${player1DiceOrder}. <br><br>${userName2}, Please press submit to roll your dice`;
+    currentGameMode = "Player 2";
+  } else if (
+    currentGameMode == "Player 1 order" &&
+    (input != 1 || input != 2)
+  ) {
+    myOutputValue = `Please choose the order of the dice.`;
+  }
+
+  // 4. player 2 roll dice
+  if (currentGameMode == "Player 2" && input == "") {
+    //player 2's turn
+    diceRoll3.push(rollDice());
+    diceRoll4.push(rollDice());
+    myOutputValue =
+      userName2 +
+      ", You rolled " +
+      diceRoll3[0] +
+      " for dice 1 and " +
+      diceRoll4[0] +
+      " for dice 2. <br><br> Please choose the order of the dice. ";
+    currentGameMode = "Player 2 order";
+  }
+
+  // 5. player 2 choose which order to roll (dice3dice4 aka input == 1)
+  else if (currentGameMode == "Player 2 order" && input == 1) {
+    var player2DiceOrder = diceRoll3[0].toString() + diceRoll4[0].toString();
+    player2Num = Number(player2DiceOrder);
+    console.log("player2Num");
+    console.log(player2Num);
+    // 6. compare the dice numbers & winner
+    if (player1Num > player2Num == true) {
+      player1score = Number(player1score) + 1;
+      player2score = Number(player2score) + 0;
+      myOutputValue = `Your number is ${player2DiceOrder}. <br><br> The winner is ${userName1}. <br><br> The current scores are:<br> ${userName1}: ${player1score}<br> ${userName2}:${player2score}`;
+      currentGameMode = "Player 1";
+    } else if (player1Num < player2Num == true) {
+      player1score = Number(player1score) + 0;
+      player2score = Number(player2score) + 1;
+      myOutputValue = `Your number is ${player2DiceOrder}. <br><br> The winner is ${userName2}. <br><br> The current scores are:<br> ${userName1}: ${player1score}<br> ${userName2}: ${player2score}`;
+      currentGameMode = "Player 1";
+    }
```

Comment:
This block of code is repeated at lines 128-138 - you might want to put this in a helper function and call the function at this 2 places to shorten your code!